src/main/groovy/com/netflix/nebula/grails/GrailsPlugin.groovy

```diff
@@ -153,26 +153,28 @@ class GrailsPlugin implements Plugin<Project> {
         configureEclipse(project)
 
         project.plugins.withType(PublishingPlugin) {
-            project.publishing {
-                publications {
-                    withType(IvyPublication) {
-                        artifact(getGrailsOutputArtifact(project)) {
-                            conf "runtime"
-                        }
-                        descriptor.withXml { XmlProvider xml ->
-                            def root = xml.asNode()
-                            def deps = root.dependencies[0]
-                            deps.@defaultconfmapping = "%->default"
-                            def runtimeClasspath = project.configurations.runtimeClasspath
-                            Map<String, String> revConstraintLookup = runtimeClasspath.allDependencies.collectEntries { Dependency dep ->
-                                [(dep.group + ':' + dep.name): dep.version]
+            project.afterEvaluate {
+                project.publishing {
+                    publications {
+                        withType(IvyPublication) {
+                            artifact(getGrailsOutputArtifact(project)) {
+                                conf "runtime"
                             }
-                            runtimeClasspath.resolvedConfiguration.firstLevelModuleDependencies.each { ResolvedDependency dep ->
-                                deps.appendNode('dependency', ['org'          : dep.moduleGroup,
-                                                               'name'         : dep.moduleName,
-                                                               'rev'          : dep.moduleVersion,
-                                                               'conf'         : 'runtime->default',
-                                                               'revConstraint': revConstraintLookup[(dep.moduleGroup + ':' + dep.moduleName)]])
+                            descriptor.withXml { XmlProvider xml ->
```

Comment:
Question for you: is this just adding the dependencies to the xml rather than just resolving them?